inngest-core/src/main/kotlin/com/inngest/ServeConfig.kt

```diff
@@ -0,0 +1,52 @@
+package com.inngest
+
+class ServeConfig(
+    val client: Inngest,
+    internal val id: String? = null,
+    internal val signingKey: String? = null,
+    internal val serveOrigin: String? = null,
+    internal val servePath: String? = null,
+    // streaming: String = "false" // probably can't stream yet
+    internal val logLevel: String? = null,
+    internal val baseUrl: String? = null,
+) {
+    fun appId(): String {
+        if (id != null) return id
+        return client.appId
+    }
+
+    fun signingKey(): String {
+        if (signingKey != null) return signingKey
+        return System.getenv(InngestSystem.EventKey.value) ?: ""
```

Comment:
s/EventKey/SigningKey

I'll make the change if you don't get to it first with your tests